### ISSUE TITLE
(Staging) possible Bing annotation or template issue 

### ISSUE THREAD

**Issue by tonyespinoza1** [stanford-oval/thingpedia-common-devices#418]
both of these cases should work.

![image](https://user-images.githubusercontent.com/86493411/128955688-3154f5ec-6aa7-4bf1-9e0a-45e070d77f6c.png)

```
====
# main/1f27104a-8183-4ae2-8c25-caa7cdee3666
#! timestamp: 2021-08-11T01:24:08.966Z
U: ask bing what is the capital of the united states
UT: $failed;
#! timestamp: 2021-08-11T01:24:08.978Z
A: Sorry, I did not understand that. Can you rephrase it?
====
# main/bbd0b847-55c8-4ee8-b33a-0a3cd7867fbb
#! timestamp: 2021-08-11T01:24:18.369Z
U: ask bing capital of united states
UT: $dialogue @org.thingpedia.dialogue.transaction.execute;
UT: @com.bing.web_search(query="capital of united states");
```

**Comment by jmhw0123** [comment]
This issue is caused by Bing only returned its answerType without returning any values in its `rankingResponse` object.

**Comment by tonyespinoza1** [comment]
The spelling I used in my example was a common misspelling (Capital vs Capitol), which is why I too assumed it was a training issue.

Interestingly, I dropped "Ask Bing" and it did go to Bing correctly.  So I will revise my request here to make this work with the correct spelling as it does with the common misspelling.

```
# main/0443689a-7d76-4b62-a944-27ab60e1264a
#! timestamp: 2021-09-09T01:30:17.462Z
U: what is the capital of the united states
UT: $dialogue @com.bing.webPages;
C: $dialogue @com.bing.webPages;
#! timestamp: 2021-09-09T01:30:17.478Z
A: Using Bing I found List of capitals in the United States - Wikipedia. Washington, D.C. has been the federal capital of the United States since 1800. Each U.S. state has its own capital city, as do many of its insular areas..
AT: @com.bing.reply;
#! timestamp: 2021-09-09T01:30:23.224Z
U: what is the capitol of the united states
UT: $dialogue @org.thingpedia.dialogue.transaction.execute;
UT: @org.thingpedia.builtin.thingengine.builtin.faq_reply(question=enum about_almond_location);
```
![image](https://user-images.githubusercontent.com/86493411/132607479-bbc66924-ef90-47dc-a47d-501c187c3777.png)

**Comment by tonyespinoza1** [comment]
@jmhw0123, i've finally got a testing environment up and running with the latest and will go through everything marked Ready for QA.  see comments above on this one.

**Comment by jmhw0123** [comment]
We should test this with "ask bing" in the query and see what happens.

**Comment by tonyespinoza1** [comment]
sorry, i should have mentioned that i did try that!  here's the proof:
![image](https://user-images.githubusercontent.com/86493411/132784898-7e3b94a7-5cf4-433a-9417-69c7d2c67fbc.png)

```
====
# main/9f909cdd-4af0-4ce0-92df-e5a81e2683a7
#! timestamp: 2021-09-10T01:37:50.841Z
U: ask bing what is the capitol of the united states
UT: $dialogue @com.bing.webPages;
C: $dialogue @com.bing.webPages;
#! timestamp: 2021-09-10T01:37:50.856Z
A: Using Bing I found United States Capitol - Wikipedia. The United States Capitol, often called The Capitol or the Capitol Building, is the meeting place of the United States Congress and the seat of the legislative branch of the U.S. federal government. It is located on Capitol Hill at the eastern end of the National Mall in Washington, D.C..
AT: @com.bing.reply;
#! timestamp: 2021-09-10T01:38:04.096Z
U: ask bing what is the capital of the united states
UT: $dialogue @com.bing.webPages;
```

**Comment by tonyespinoza1** [comment]
to be clear i think we should map Capitol to Capital in this context:

here's the reply from google:
https://www.google.com/search?q=what+is+the+capitol+of+the+united+states&rlz=1C5CHFA_enUS964US964&oq=what+is+the+capitol+of+the+united+states&aqs=chrome..69i64j0i10l5j0i10i22i30j0i22i30j0i10i22i30j0i22i30.7944j1j1&sourceid=chrome&ie=UTF-8

![image](https://user-images.githubusercontent.com/86493411/132785236-dae8cb00-955d-46a3-8e26-a495b8189bab.png)
